pkgs/blast_repo/bin/blast_repo.dart

```diff
@@ -47,7 +48,7 @@ Future<void> main(List<String> args) async {
     return;
   }
 
-  if (argResults['help'] as bool) {
+  if (argResults['help'] as bool || argResults.rest.isEmpty) {
```

Comment:
huh? we don't support `rest` do we?

Reply:
We use it for the `slug`, below (if you run this command w/o any args, you get an NPE on line 56).

Reply:
oh RIGHT